instructure/canvas-ios
# Native quiz preview

There's a slight change in the behaviour compared to the react version: upon loading the webview with the quiz preview url the react version automatically started the quiz submission. I couldn't easily reproduce the same so now the user has to start the preview inside the webview which will result in 1 or 2 extra taps.

refs: MBL-16290
affects: Teacher
release note: none

test plan:
- Enable native teacher quiz feature flag
- Go to quizzes, select a quiz
- On the quiz details page tap on preview quiz button
- Observe loading indicator, quiz preview load, modal presentation styles.

## Checklist
